src/api/svg.ts

```diff
@@ -156,10 +158,17 @@ const runnersToPage = (
     const anchor = element.svgAttributes.textAnchor;
     const text = element.childNodes[0].text;
     const fontSize = element.svgAttributes.fontSize || 12;
-    const font =
-      options.fonts && element.svgAttributes.fontFamily
-        ? options.fonts[element.svgAttributes.fontFamily]
-        : undefined;
+    const fontStyle = element.svgAttributes.fontStyle
+      ? `_${element.svgAttributes.fontStyle}`
+      : '';
+    const fontWeight = element.svgAttributes.fontWeight
+      ? `_${element.svgAttributes.fontWeight}`
+      : '';
+    const svgFont = element.svgAttributes.fontFamily
```

Comment:
If a "styled" font is not provided in the list of fonts, you need to fallback to an equivalent.

font_bold_italic should fallback to font_bold or font_italic or font
font_bold should fallback to font
font_italic should fallback to font.